src/docs/product/integrations/source-code-mgmt/github/index.mdx

```diff
@@ -37,7 +37,7 @@ Sentry owner, manager, or admin permissions, and GitHub owner permissions are re
 
 The GitHub integration is available for all projects under your Sentry organization. You can connect multiple GitHub organizations to one Sentry organization, but you **cannot** connect a single GitHub organization to multiple Sentry organizations.
 
-### GitHub Enterprise
+# GitHub Enterprise
```

Comment:
```suggestion
## GitHub Enterprise
```

If you make this an L1 heading, it won't show up in the docs sidebar:
<img width="202" alt="Screenshot 2023-04-19 at 10 22 35 AM" src="https://user-images.githubusercontent.com/3520671/233152328-66582406-76a6-494e-b5c0-6343f15a2a85.png">